app/controllers/activities_controller.rb

```diff
@@ -0,0 +1,36 @@
+class ActivitiesController < ApplicationController
+	include ActivitiesHelper
+	before_filter :redirect_to_root_if_not_logged_in
```

Comment:
That's nice :p

On Wed, May 20, 2015 at 4:12 PM, B V Satyaram <notifications@github.com>
wrote:

> In app/controllers/activities_controller.rb
> <https://github.com/chi6rag/HourGlass/pull/1#discussion_r30689125>:
>
> > @@ -0,0 +1,36 @@
> > +class ActivitiesController < ApplicationController
> > +	include ActivitiesHelper
> > +	before_filter :redirect_to_root_if_not_logged_in
>
> :authenticate_user! is a better name
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/chi6rag/HourGlass/pull/1/files#r30689125>.
>